src/main/java/ch/njol/skript/expressions/ExprElement.java

```diff
@@ -45,10 +47,14 @@
 public class ExprElement extends SimpleExpression<Object> {
 
 	static {
-		Skript.registerExpression(ExprElement.class, Object.class, ExpressionType.PROPERTY, "(-1¦[the] first|1¦[the] last|0¦[a] random|2¦%-number%(st|nd|rd|th)) element [out] of %objects%");
+		Skript.registerExpression(ExprElement.class, Object.class, ExpressionType.PROPERTY, "(0:[the] first|1¦[the] last|2:[a] random|3:%-number%(st|nd|rd|th)) element [out] of %objects%");
```

Comment:
Was this left intentional or was it missed?
```suggestion
		Skript.registerExpression(ExprElement.class, Object.class, ExpressionType.PROPERTY, "(0:[the] first|1:[the] last|2:[a] random|3:%-number%(st|nd|rd|th)) element [out] of %objects%");
```